README.md

```diff
@@ -1,12 +1,18 @@
 # @brave-intl/currency
 
-a currency conversion ratio cache
+a currency conversion ratio cache. 
+Atual version is 1.4.6 
```

Comment:
no need to mention version in the readme, package.json should be enough to tip people off.